ui/app/components/action-card.hbs

```diff
@@ -71,7 +71,12 @@
       <code><pre>Error: {{this.instance.error}}</pre></code>
     {{/if}}
     {{#if this.instance.messages.length}}
-      <code><pre>{{this.instance.messages}}</pre></code>
+      <code tabindex="0">
```

Comment:
Side-effect: now you can focus in the action output and do things like `pgdn` etc.